examples/newapi_localizer_analysis.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Same comment here. The example should inly use the best API, so please remove plot_localizer_analysis.py and rename newapi_localizer_analysis.py to plot_localizer_analysis.py.